src/_pytest/compat.py

```diff
@@ -195,72 +166,50 @@ def _translate_non_printable(s):
     return s.translate(_non_printable_ascii_translate_table)
 
 
-if _PY3:
-    STRING_TYPES = bytes, str
-    UNICODE_TYPES = six.text_type
+STRING_TYPES = bytes, str
+UNICODE_TYPES = six.text_type
 
-    if PY35:
+if PY35:
```

Comment:
This should be fixed, PY35 is `sys.version_info[:2] >= (3, 5)` so this is always `True`